reports/report.md

```diff
@@ -99,7 +96,7 @@ High level information about the dataset variables can be found below:
 | 13    | Browser                  | Internet browser used by the visitor                                                                                                                           |
 | 14    | Region                   | Demographic region when the user accessing the site                                                                                                            |
 | 15    | TrafficType              | From which source the visitor arrived at the site                                                                                                              |
-| 16    | VisitorType              | Visitor type as “New Visitor”, “Returning Visitor”, and “Other”                                                                                                |
+| 16    | VisitorType              | Visitor type as “New Visitor,” “Returning Visitor,” and “Other”                                                                                                |
```

Comment:
Just a very minor thing: comma(,) should be placed after "

Reply:
I agree.. this was auto formatted when I save unfortunately. I will look into why it is happening